sechub-solution/docker/SecHub-Debian.dockerfile

```
@@ -63,6 +68,7 @@ RUN mkdir --parent "$BUILD_FOLDER" && \
     cd "sechub" && \
     "./buildExecutables" && \
     if [[ -n "$TAG" ]]; then git checkout tags/"$TAG" -b "$TAG"; fi && \
+    if [[ -n "$BRANCH" ]]; then git checkout "$BRANCH"; fi && \
     cp "sechub-server/build/libs/sechub-server-0.0.0.jar" --target-directory "$SECHUB_ARTIFACT_FOLDER"
```

When building a tagged version, the jar will not be named `0.0.0`.
Suggestion:
```suggestion
    cp "sechub-server/build/libs/sechub-server-*.jar" --target-directory "$SECHUB_ARTIFACT_FOLDER"
```